sdcm/cluster_k8s/__init__.py

```diff
@@ -328,6 +331,8 @@ def __init__(self, params: dict, user_prefix: str = '', region_name: str = None,
         ]
         self._scylla_cluster_events_threads = {}
         self.chaos_mesh = ChaosMesh(self)
+        self.racks_count = len(self.params.get("availability_zone").split(",")
+                               ) if self.params.get("availability_zone") else 1
```

Comment:
How can I count racks on this step?

Reply:
By reading the `rack_name` of each of the DB pods/nodes.

Reply:
it's hard coded ...

```
self.rack_name = "rack-1"
```

and no code is changing it.

I think for now it's safe to assume k8s setup has one rack, until someone really implement it

Reply:
> it's hard coded ...
> 
> ```
> self.rack_name = "rack-1"
> ```
> 
> and no code is changing it.
> 
> I think for now it's safe to assume k8s setup has one rack, until someone really implement it

My comment is about using the new `rack_name` attr added in this PR for the PodContainer class (the one where pod info gets read), not K8S one.

Reply:
I added it here because of provision-k8s-local-kind-aws failure
https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-7310/12/execution/node/160/log/

```
 File "/home/ubuntu/scylla-cluster-tests/sdcm/cluster_k8s/mini_k8s.py", line 437, in host_node
   node.init()
 File "/home/ubuntu/scylla-cluster-tests/sdcm/cluster.py", line 359, in init
   self.log = SDCMAdapter(self.log, extra={"prefix": str(self)})
 File "/home/ubuntu/scylla-cluster-tests/sdcm/cluster.py", line 999, in __str__
   self._dc_info_str())
 File "/home/ubuntu/scylla-cluster-tests/sdcm/cluster.py", line 986, in _dc_info_str
   if self.parent_cluster.racks_count > 1 and self.node_rack:
 AttributeError: 'LocalKindCluster' object has no attribute 'racks_count'
```

It complains on `LocalKindCluster`

Reply:
Looks like only one node in this cluster (`LocalMinimalClusterBase`). Is it correct?

Reply:
Local K8S cluster has many nodes.
K8S cluster should not have any racks info.
K8S cluster may be multitenant and each tenant may have different set of racks.
FOr getting rack info must be used only Pod/Container classes.